index.html

```diff
@@ -168,6 +168,10 @@ <h2>MiniApp Global Application Lifecycle States</h2>
           Lifecycle state for MiniApp [=Document/in error=]. It is triggered once the MiniApp is confronted with script error.
         </dd>
       </dl>
+            
+         <p>
+            The mapping to existing Web specifications such as <a href="https://www.w3.org/TR/service-workers/">Service Workers</a> is listed in <a href="https://github.com/w3c/miniapp-lifecycle/blob/main/docs/explainer.md#comparison-with-some-related-work-in-w3c-such-as-service-worker-page-visibility-and-page-lifecycle">MiniApp Lifecycle Explainer</a>. 
```

Comment:
As I said in https://github.com/w3c/miniapp-lifecycle/issues/15#issuecomment-925673970 , the "mapping" is to demonstrate how to polyfill the behavior of MiniApp Lifecycle using existing Web APIs. The "mapping" in the explainer may be considered too rough.

But at the same time, our [charter](https://www.w3.org/2021/01/miniapps-wg-charter.html#rec-track) says "**whenever possible**, the specification should provide a mapping to existing Web specifications such as [Service Workers](https://www.w3.org/TR/service-workers/) and [Page Visibility](https://w3c.github.io/page-visibility/)", so it is not a MUST.

Reply:
The "mapping" comment from TAG is about the explainer (https://github.com/w3ctag/design-reviews/issues/523#issuecomment-928975772), which has been somewhat accepted by TAG (https://github.com/w3ctag/design-reviews/issues/523#issuecomment-1032411658). Therefore, besides updating the explainer, I also add some reference in Lifecycle spec.

Or, we can simply choose to only update the explainer, not the spec. What do you think?

Reply:
Works for me.

Reply:
Consensus to merge in MiniApp WG meeting 2022-02-24